script/setup

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# script/setup: Set up application for the first time after cloning, or set it
+#               back to the initial first unused state.
+
+set -e
+cd "$(dirname "$0")/.."
+
+
+script/bootstrap
+script/-check-docker-compose
+
+echo
+echo "==> Downloading and restoring test data into mongodb…"
+curl 'https://boutell.dev/pa-wildflower-selector/pa-wildflower-selector.archive.gz' \
+     | gunzip -c \
+     | docker-compose exec -T mongodb bash -c '
+        mongorestore \
+            --drop \
+            --archive \
+            --username=${MONGO_INITDB_ROOT_USERNAME} \
```

Comment:
This has a username and a port but I'm not sure it's talking to mongo running on the same server address as it does in the app.

Reply:
well we were getting a different error before this step was put in place so it seemed to have done _something_ the app sees

Reply:
I will give it a try as soon as I am able. I gather @ZacharyLeahan is looking into it too.

Reply:
I didn't see the mongorestore command in your script setting the database name anywhere, is it embedded in the dump? is the problem that we configured the "new" app name as the DB within the app connection string instead of sticking with the existing name?